apps/meteor/app/2fa/server/code/EmailCheck.spec.ts

```diff
@@ -0,0 +1,79 @@
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+import proxyquire from 'proxyquire';
+import sinon from 'sinon';
+
+const isOAuthUserMock = sinon.stub();
+const settingsMock = sinon.stub();
+
+const { EmailCheck } = proxyquire.noCallThru().load('./EmailCheck', {
+	'@rocket.chat/core-typings': {
+		isOAuthUser: isOAuthUserMock,
```

Comment:
You shouldn't need to mock this func, apart from not affecting, the checks it performs are valuable for what you're testing here.

You should pass data that will make this func fail/pass instead

Reply:
You are right, fixed and pushed.